core/templates/pages/splash-page/splash-page.component.ts

```diff
@@ -77,91 +68,60 @@ export class SplashPageComponent implements OnInit {
     );
   }
 
-  private _nagivateToClassroomPage(): void {
+  private _navigateToClassroomPage(): void {
     this.windowRef.nativeWindow.location.href = this.classroomUrl;
   }
 
+  private _navigateToAccountCreationPage(): void {
+    this.userService.getLoginUrlAsync().then(loginUrl => {
+      if (loginUrl) {
+        setTimeout(() => {
+          this.windowRef.nativeWindow.location.href = loginUrl;
+        }, 150);
+      } else {
+        this.windowRef.nativeWindow.location.reload();
```

Comment:
And why reload here?

Reply:
This reload thing is there throughout the codebase where we navigate to Google 'sign-in' page.